omegaconf/base.py

```diff
@@ -400,7 +401,9 @@ def _resolve_interpolation_from_parse_tree(
         parse_tree: OmegaConfGrammarParser.ConfigValueContext,
         throw_on_resolution_failure: bool,
     ) -> Optional["Node"]:
```

Comment:
Please add docstring to this function.
In particular when it's returning None.

Reply:
Done in3d350db